.github/workflows/book.yml

```diff
@@ -48,11 +47,11 @@ jobs:
 
       - run: mdbook build -d public
         working-directory: book
-      - run: echo "DEST=$(if [ "$GITHUB_EVENT_NAME" == "release" ] || [ "$GITHUB_EVENT_NAME" == "workflow_dispatch" ]; then echo 'stable/latest'; else echo 'git'; fi)" >> ${GITHUB_ENV}
+      - run: echo "DEST='stable/latest'" >> ${GITHUB_ENV}
```

Comment:
I suppose this change is not intended? or do you want to drop the git variant of the book?

Reply:
Yes. I see no reason to have a git variant of the book that depends on stable gtk4-rs

Reply:
Well the git variant could be wip chapters and things like that. If that is the decision, the commit should also drop the links to the git version of the book and probably do some re-direction between old (git) links to stable/latest

Reply:
> Well the git variant could be wip chapters 

I'd like the option to give reviewers a rendered version of a PR. I'll experiment if that can be done with a workflow dispatch.

Reply:
The current behaviour is:
- stable: used for every release or push to master
- git: used when using workflow dispatch